ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/MessageScrubberTest.java

```diff
@@ -271,6 +271,20 @@ public void testBug1162183Affected() throws Exception {
         .putAll(baseAttributes).put(Attribute.APP_VERSION, "41.0.2").build()));
   }
 
+  @Test
+  public void testBug1642386Affected() throws Exception {
+    Map<String, String> baseAttributes = ImmutableMap.<String, String>builder()
+        .put(Attribute.DOCUMENT_NAMESPACE, ParseUri.TELEMETRY).put(Attribute.DOCUMENT_TYPE, "sync")
+        .build();
+
+    assertFalse(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
+        .putAll(baseAttributes).put(Attribute.APP_VERSION, "60.6.1").build()));
+    assertTrue(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
+        .putAll(baseAttributes).put(Attribute.APP_VERSION, "25.0.1").build()));
+    assertTrue(MessageScrubber.bug1642386Affected(ImmutableMap.<String, String>builder()
+        .putAll(baseAttributes).put(Attribute.APP_VERSION, "26.1").build()));
```

Comment:
Can you add a test case for "10.25.1"? I don't know that we ever have minor or patch versions as high as 25, but I want to make sure the regex wouldn't accidentally match it.